.circleci/monorepo.sh

```diff
@@ -147,11 +147,19 @@ function print_status {
 }
 
 function create_request_body {
-  echo "$1" | 
-  jq --raw-output --arg branch "${CIRCLE_BRANCH}" --arg trigger "${TRIGGER_PARAM_NAME}" --argjson params "${CI_PARAMETERS:-null}" '. | 
-    map(select(.changes > 0)) | 
-    reduce .[] as $i (($params // {}) * { ($trigger): false }; .[$i.package] = true) | 
-    { branch: $branch, parameters: . } | 
+  echo "$1" |
+  jq --raw-output --arg branch "${CIRCLE_BRANCH}" --arg trigger "${TRIGGER_PARAM_NAME}" --argjson params "${CI_PARAMETERS:-null}" '. |
+    map(select(.changes > 0)) |
+    reduce .[] as $i (($params // {}) * { ($trigger): false }; .[$i.package] = true) |
+    { branch: $branch, parameters: . } |
+    @json'
+}
+
+function create_request_tag {
```

Comment:
Added this function specific to building the json blog that is submitted to trigger the circleci job. Needed 'tag' instead of branch.